mitosheet/src/utils/actions.tsx

```diff
@@ -572,7 +574,12 @@ export const createActions = (
                 setEditorState(undefined);
 
                 // Open slack
-                window.open(SLACK_INVITE_LINK, '_blank')
+                if (userProfile.mitoConfig['support_email'] === DEFAULT_SUPPORT_EMAIL) {
```

Comment:
Def move to enum everywhere. I want to be able to find and replace w/ 100% accuracy :-)